src/library_webgpu.js

```diff
@@ -1690,14 +1807,29 @@ var LibraryWebGPU = {
     pipeline.label = UTF8ToString(labelPtr);
   },
 
+  // wgpuBindGroup
+
+  wgpuBindGroupSetLabel: function(bindGroupId, labelPtr) {
+    var bindGroup = WebGPU.mgrBindGroup.get(bindGroupId);
+    bindGroup.label = UTF8ToString(labelPtr);
+  },
+
+  // wgpuBindGroupLayout
+
+  wgpuBindGroupLayoutSetLabel: function(bindGroupLayoutId, labelPtr) {
+    var bindGroupLayout = WebGPU.mgrBindGroupLayout.get(bindGroupLayoutId);
+    bindGroupLayout.label = UTF8ToString(labelPtr);
+  },
+
   // wgpuBuffer
 
+  // In webgpu.h offset and size are passed in as size_t.
+  // And we assume that size_t is always 32bit in emscripten.
```

Comment:
```suggestion
  // And library_webgpu assumes that size_t is always 32bit in emscripten.
```

Some parts of Emscripten are getting updates to work with WASM64, so just clarifying